composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "dev-utility_separation",
+        "php-http/httplug": "dev-master@dev",
```

Comment:
message-factory has a branch-alias for 0.2. unless @sagikazarmark wants to bump that to 1.0, you could change to `~0.2@dev`